www/src/css/layout.css

```diff
@@ -37,3 +35,8 @@
   width: 100%;
   z-index: 10000;
 }
+
+.grey-wrapper {
```

Comment:
I named the class in line with `inner-wrapper` which comes from Vanilla but for consistency with the existing CSS adding the `b-` prefix would make sense.

The two `-wrapper` classes can't be used on the same element unfortunately: `inner-wrapper` sets a `max-width` and centres itself in the viewport while `grey-wrapper` is used to achieve a full-width grey background that the design calls for on pages with snap lists.